rules/windows/process_creation/proc_creation_win_werfault_reflect_debugger_exec.yml

```diff
@@ -0,0 +1,27 @@
+title: Potential ReflectDebugger Content Execution Via WerFault.EXE
+id: fabfb3a7-3ce1-4445-9c7c-3c27f1051cdd
+related:
+    - id: 0cf2e1c6-8d10-4273-8059-738778f981ad
+      type: derived
+status: experimental
+description: Detects execution of "WerFault.exe" with the "-pr" commandline flag that is used to run files stored in ReflectDebugger key which could be store malware path
+references:
+    - https://cocomelonc.github.io/malware/2022/11/02/malware-pers-18.html
+    - https://www.hexacorn.com/blog/2018/08/31/beyond-good-ol-run-key-part-85/
+author: X__Junior (Nextron Systems)
+date: 2023/06/30
+tags:
+    - attack.execution
+logsource:
+    product: windows
+    category: process_creation
+detection:
+    selection_img:
+        - Image|endswith: '\WerFault.exe'
+        - OriginalFileName: 'WerFault.exe'
+    selection_cli:
+        CommandLine|contains: ' -pr '
```

Comment:
can the flags be used in different ways?

Reply:
If you mean like with a "/", It doesn't seem to be accepted.

Reply:
I thought more about `-p -r`. But some more googling shows me `-pr` is one flag, not two concatenated. Sadly it's not well documented but there are 3rd party explanations on the web.